### DISCUSSION TITLE
Github copilot no longer able to solve problems it used to solve

### DISCUSSION THREAD

**Discussion by Amagash** [community/community#11735] [Copilot Conversations]
On the page https://copilot.github.com/ there is a demonstration of copilot suggesting the code for the parse_expense function

![image](https://user-images.githubusercontent.com/20380447/154750181-7318ca5a-8d10-4d86-ba29-4af1fa0c457d.png)

It is no longer able to give a suggestion for that problem. I noticed this change on Feb 3rd. 

I used to give demos of copilot on other problems like this one such as the "Two sum" problem or a "Sudoku solver" and again it's no longer able to suggest a block of code for those problems. 

![image](https://user-images.githubusercontent.com/20380447/154751540-082cb95a-1766-4ac0-9037-bbcae6b4a9a5.png)

**Comment by ghost** [comment]
import datetime

def parse_expreses(expense_string):
  """Parse the list of expenses and return the list of triples (date, value, currency).
  Ignore lines starting with #.
  Parse the date using datetime.
  Example exprenses_string:
    2016-01-02 -34.01 USD
    2016-01-3 2.69 DKK
    2016-01-03 -2.72 EUR
  """  
  expenses = []
  for line in expense_string.splitlines():
    if line.startswith('#'):
      continue
    date, value, currency = line.split()
    expenses.append((datetime.datetime.strptime(date, '%Y-%m-%d'), float(value), currency))
  return expenses

**Reply by davecheney** [reply]
@eatcosmos 
i'm sorry you are having difficulties. Looking at your telemetry it looks like you are making and receiving completions. Can you please tell me a bit more about what you tried, and what happened. Thank you.

**Reply by ghost** [reply]
> @eatcosmos i'm sorry you are having difficulties. Looking at your telemetry it looks like you are making and receiving completions. Can you please tell me a bit more about what you tried, and what happened. Thank you.

test this function demos
![image](https://user-images.githubusercontent.com/83876321/186726037-454ac84d-c372-4b3c-acb3-6ada2e5526e2.png)

**Reply by davecheney** [reply]
Thank you for your reply. Are you happy that copilot is working?

**Reply by ghost** [reply]
> Thank you for your reply. Are you happy that copilot is working?

yes, hope it will be more clever in the future, natural language/base on comment programming shuld be the future.
I think copilot maybe do with gpt-3, according to article to generate the whole executable software
